src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ReblockGVCFIntegrationTest.java

```diff
@@ -185,4 +185,19 @@ public void testMQHeadersAreUpdated() throws Exception {
             }
         }
     }
+
+    @Test
+    public void testReReblocking() {
+        final File input = new File(getToolTestDataDir() + "alreadyReblocked.chr22snippet.vcf");
+        final File output = createTempFile("rereblockedgvcf", ".vcf");
+        final ArgumentsBuilder args = new ArgumentsBuilder();
+        args.add("V", input)
+                .addOutput(output);
+        runCommandLine(args);
+
+        Pair<VCFHeader, List<VariantContext>> inputVCs = VariantContextTestUtils.readEntireVCFIntoMemory(input.getAbsolutePath());
+        Pair<VCFHeader, List<VariantContext>> outputVCs = VariantContextTestUtils.readEntireVCFIntoMemory(output.getAbsolutePath());
+
+        Assert.assertTrue(inputVCs.getRight().size() > outputVCs.getRight().size());
```

Comment:
This test seems a bit weak.... can you do a more through test e.g. the unaffected blocks should be in the same order and unchanged... etc.